plugins/arDominionB5Plugin/templates/_layout_start.php

```diff
@@ -8,6 +8,8 @@
     <link rel="shortcut icon" href="<?php echo public_path('favicon.ico'); ?>"/>
     <!-- TODO: Move other CSS to SCSS build (some not possible due to https://github.com/sass/sass/issues/2535) -->
     <link rel="stylesheet" href="/node_modules/mediaelement/build/mediaelementplayer.min.css">
+    <link rel="stylesheet" href="/node_modules/jquery-ui-dist/jquery-ui.min.css">
+    <link rel="stylesheet" href="/node_modules/jstree/dist/themes/default/style.min.css">
```

Comment:
Do they use relative paths like mediaelementplayer? Or could they be moved to the SCSS build?

Reply:
These packages do not include scss files - can they be moved to the SCSS build if they do not @jraddaoui ?

Reply:
I have moved jquery-ui to main.scss but jstree has references to images that fail when bundled with scss.